src/main/java/com/kodilla/ecommercee/CartController.java

```diff
@@ -1,4 +1,44 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.CartDto;
+import com.kodilla.ecommercee.mapper.CartMapper;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/V1/cart")
 public class CartController {
+    private final CartMapper cartMapper;
+
+    public CartController(CartMapper cartMapper) {
+        this.cartMapper = cartMapper;
+    }
+
+    @GetMapping(value = "getProductsInCart")
+    public Map<Long, Integer> getProductsInCart(long cartId) throws CartNotFoundException {
+        return new HashMap<Long, Integer>();
+    }
+
+    @PutMapping(value = "updateCart")
+    public CartDto updateCart(CartDto cartDto, long productId, int quantity) {
+        Map<Long, Integer> productsInCart = new HashMap<>(2, 5);
+        return new CartDto(1L, productsInCart);
+    }
+
+    @RequestMapping(method = RequestMethod.DELETE, value = "deleteProductFromCart")
+    public void deleteProductFromCart(long cartId, long productId) {
+
+    }
+
+    @PostMapping(value = "orderProducts")
+    public void orderProducts() {
```

Comment:
ta metoda powinna byc w kontrolerze dotyczacym produktow. Jest do usuniecia

Reply:
Nie rozumiem, ponieważ w specyfikacji projektu z modułu 21.5 napisane jest:
Lista endpointów:
carts:
-utworzenie pustego koszyka
-pobranie elementów z pustego koszyka
-dodanie elementów (produktów) do koszyka
-usunięcie konkretnego produktu z koszyka
-utworzenie zamówienia na podstawie koszyka
Czy chodzi ci o to abym zamienił orderProducts na createOrder?